### ISSUE TITLE
Yellow focus-ring outlines (and yellow background on `pressed` state) unexpectedly appear on UI buttons in Tray and Navigation Bar (on WaveVR)

### ISSUE THREAD

**Issue by cvan** [MozillaReality/FirefoxReality#1005]
## Hardware
[Pico G2](https://www.pico-interactive.com/g2) (WaveVR)

## Steps to Reproduce
1. Press the URL bar.
2. Once the keyboard opens, press the × button above the keyboard to dismiss the keyboard.
3. Notice a thin, yellow ring outline appears around the **Help** icon in the Tray.

## Current Behavior
On WaveVR but not on Oculus, it appears that the **Help** icon element receives an incorrect focus event (resulting in a thin, yellow ring outline around the element) when a different UI element is dismissed.

## Expected Behavior
Upon pressing another UI element (e.g., dismissing the keyboard), the **Help** icon receives an incorrect focus event (resulting in a thin, yellow ring outline around the element) when a different UI item is dismissed.

![image](https://user-images.githubusercontent.com/203725/54266290-5351da80-4534-11e9-983d-c3ff763e5839.png)
![2019-03-13_01:23:37](https://user-images.githubusercontent.com/203725/54266448-a592fb80-4534-11e9-9217-33a21b90abcb.png)
![2019-03-13_01:23:48](https://user-images.githubusercontent.com/203725/54266449-a592fb80-4534-11e9-8117-a22b9ce74be9.png)
![2019-03-13_01:23:57](https://user-images.githubusercontent.com/203725/54266451-a592fb80-4534-11e9-861f-9825cd96c597.png)
![2019-03-13_01:24:28](https://user-images.githubusercontent.com/203725/54266452-a62b9200-4534-11e9-8f79-98315e73796f.png)
![](https://user-images.githubusercontent.com/203725/54264177-b8ef9800-452f-11e9-9ea9-0a2682e7afa9.png)
![2019-03-13_01:24:40](https://user-images.githubusercontent.com/203725/54266453-a62b9200-4534-11e9-829a-03adbffc16b6.png)
![2019-03-13_01:24:54](https://user-images.githubusercontent.com/203725/54266454-a62b9200-4534-11e9-9b8a-41e3ea2a45ee.png)

**Comment by bluemarvin** [comment]
I am unable to reproduce on Pico G2 with [master](https://github.com/MozillaReality/FirefoxReality/commit/e475629fc9b51a1c0e95a6cbd57d412d7610261d)

**Comment by philip-lamb** [comment]
OK, thanks for the attempts to reproduce.

**Comment by cvan** [comment]
This is reproducible again on `master` (78508515) and `release-1.2` (2566225).

You can also see a gnarly yellow background-colour change too if you press-and-hold the trigger button on any UI buttons (e.g., those in the Navigation Bar, the Tray, buttons in the Settings sub-menus, etc.).

![image](https://user-images.githubusercontent.com/203725/57773586-36b95680-76cd-11e9-80c9-8f3d6c9e9a2f.png)

![image](https://user-images.githubusercontent.com/203725/57773570-2acd9480-76cd-11e9-99e0-256c88e9e829.png)



![image](https://user-images.githubusercontent.com/203725/57773570-2acd9480-76cd-11e9-99e0-256c88e9e829.png)

![image](https://user-images.githubusercontent.com/203725/57773580-33be6600-76cd-11e9-9c69-65f14067dd21.png)

![image](https://user-images.githubusercontent.com/203725/57773597-3a4cdd80-76cd-11e9-9821-25e68beca1e6.png)

![image](https://user-images.githubusercontent.com/203725/57773623-4769cc80-76cd-11e9-81fe-174f1ed28bbc.png)

**Comment by cvan** [comment]
I'll piggyback this widget outline onto this issue too.

@MortimerGoro reported in https://github.com/MozillaReality/FirefoxReality/pull/1218#issuecomment-492822304:
> @cvan it seems something related to accessibility or focus indicator. @bluemarvin has seen that outline in NoAPI a lot with different UIs. We can use that to reproduce it in a easier way.

on my Pico G2 (WaveVR), I also discovered a yellow outline on the widget for the Settings sub-menus, such as **Settings > Controller Options**:

![image](https://user-images.githubusercontent.com/203725/57816171-36ea3e00-772f-11e9-8f3e-89d9d02ade30.png)

(originally reported in https://github.com/MozillaReality/FirefoxReality/pull/1218#pullrequestreview-237780553.)

**Comment by philip-lamb** [comment]
Was this only on the Pico G2 or also on the Vive Focus? I've been right through the debug and accessibility options for the Vive Focus and I can't see anything obvious that would enable this.

**Comment by cvan** [comment]
> Was this only on the Pico G2 or also on the Vive Focus? I've been right through the debug and accessibility options for the Vive Focus and I can't see anything obvious that would enable this.

I'm able to reproduce on the Pico G2. Did a clean factory reset too.

Last time I checked on the Focus, I also reproduced it, but that was probably over a week or so ago. Both of my Vive Focus devices are being borrowed by Rebecca (QA) and temporarily Janice (for demoing/testing).

But tomorrow Rebecca and I are meeting in SF to go over testing in person, so we can test this bug together on the Focus tomorrow, and I'll update here our findings.
